addons/website/static/src/interactions/video/media_video.js

```diff
@@ -55,7 +54,7 @@ export class MediaVideo extends Interaction {
         // 'data-oe-expression' one (the latter is used as a workaround in 10.0
         // system but should obviously be reviewed in master).
 
-        let src = escape(this.el.getAttribute('oe-expression') || this.el.getAttribute('src'));
```

Comment:
Note: I don't see the point of escaping something set with `setAttribute` and `dataset` (through `manageIframeSrc`). Do I miss something?